resources/index.html

```diff
@@ -83,7 +83,8 @@
 		* [Game Math](https://gamemath.com/)
 		* [A Comparison of Modern Graphics APIs](https://alain.xyz/blog/comparison-of-modern-graphics-apis)
 		* [Scratchapixel](https://www.scratchapixel.com)
-		* [The Graphics Codex](https://graphicscodex.com/)
+		* [The Graphics Codex](https://graphicscodex.com/
+		* [Physically Based Rendering: From Theory To Implementation](https://www.pbr-book.org/)
```

Comment:
Maybe same here as well, for all the things covering PBR create a new header, like you did for IBL